README.md

```diff
@@ -735,6 +735,7 @@ _Data stores with expiring records, in-memory distributed data stores, or in-mem
 - [pudge](https://github.com/recoilme/pudge) - Fast and simple key/value store written using Go's standard library.
 - [regatta](https://github.com/jamf/regatta) - Fast, simple, geo-distributed KV store built for cloud native era.
 - [rosedb](https://github.com/roseduan/rosedb) - An embedded k-v database based on LSM+WAL, supports string, list, hash, set, zset.
+- [rotom](https://github.com/xgzlucario/rotom) - A tiny Redis server build with Golang, compatible with RESP protocols.
```

Comment:
Typo: 

A tiny Redis server build with Golang, compatible with RESP protocols.

```diff
-  build.
+  built.
```

Reply:
thanks for advice.